lib/si-layer-cache/src/db.rs

```diff
@@ -116,8 +120,18 @@ where
                 compute_executor.clone(),
                 tracker.clone(),
                 token.clone(),
-                30,
-                30
+                25,
```

Comment:
@sprutton1 is there anything we should think about when it comes to resizing these slices in production?

Reply:
Discussed offline